### PR TITLE
Fix the borken link in kubelet-integration.md page

### PR DESCRIPTION
**Pull request by DanyC97**
Fixes https://github.com/kubernetes/website/issues/12859

### CODE REVIEW

File: content/en/docs/setup/independent/kubelet-integration.md [hunk 1]
```diff
@@ -193,10 +193,10 @@ The DEB and RPM packages shipped with the Kubernetes releases are:
 
 | Package name | Description |
 |--------------|-------------|
-| `kubeadm`    | Installs the `/usr/bin/kubeadm` CLI tool and [The kubelet drop-in file(#the-kubelet-drop-in-file-for-systemd) for the kubelet. |
+| `kubeadm`    | Installs the `/usr/bin/kubeadm` CLI tool and [the kubelet drop-in file](#the-kubelet-drop-in-file-for-systemd) for the kubelet. |
 | `kubelet`    | Installs the `/usr/bin/kubelet` binary. |
 | `kubectl`    | Installs the `/usr/bin/kubectl` binary. |
 | `kubernetes-cni` | Installs the official CNI binaries into the `/opt/cni/bin` directory. |
-| `cri-tools` | Installs the `/usr/bin/crictl` binary from [https://github.com/kubernetes-incubator/cri-tools](https://github.com/kubernetes-incubator/cri-tools). |
+| `cri-tools` | Installs the `/usr/bin/crictl` binary from [cri-tools git repo](https://github.com/kubernetes-incubator/cri-tools). |
```
Comment (neolit123) [inline content/en/docs/setup/independent/kubelet-integration.md]: please change:
`[cri-tools git repo]` -> `the [cri-tools git repository]`

File: content/en/docs/setup/independent/kubelet-integration.md [hunk 2]
```diff
@@ -193,10 +193,10 @@ The DEB and RPM packages shipped with the Kubernetes releases are:
 
 | Package name | Description |
 |--------------|-------------|
-| `kubeadm`    | Installs the `/usr/bin/kubeadm` CLI tool and [The kubelet drop-in file(#the-kubelet-drop-in-file-for-systemd) for the kubelet. |
+| `kubeadm`    | Installs the `/usr/bin/kubeadm` CLI tool and [the kubelet drop-in file](#the-kubelet-drop-in-file-for-systemd) for the kubelet. |
```
Comment (neolit123) [inline content/en/docs/setup/independent/kubelet-integration.md]: `[the kubelet drop-in file]` -> `the [kubelet drop-in file]`

Comment (DanyC97) [reply content/en/docs/setup/independent/kubelet-integration.md]: done

File: content/en/docs/setup/independent/kubelet-integration.md (hunk 1 above)
Comment (DanyC97) [reply content/en/docs/setup/independent/kubelet-integration.md]: done

**Comment by DanyC97** [comment]
@neolit123 pls help review again. thank you!

**Comment by DanyC97** [comment]
@zacharysarah who might be the sig-docs person who can approve the PR following the review from @neolit123  ?

**Pull request merged**